resources/tagtypes/4B.json

```
@@ -0,0 +1,25 @@
+{
+	"version": 1,
+	"name": "M3 2.2\" BWRY",
+	"width": 296,
+	"height": 160,
+	"rotatebuffer": 3,
+	"bpp": 2,
+	"colortable": {
+		"white": [ 255, 255, 255 ],
+		"black": [ 0, 0, 0 ],
+		"red": [ 255, 0, 0 ],
+		"yellow": [ 255, 255, 0 ]
+	},
+	"perceptual": {
+		"white": [ 255, 255, 255 ],
+		"black": [ 0, 0, 0 ],
+		"red": [ 255, 0, 0 ],
+		"yellow": [ 200, 200, 0 ]
+	},
+	"shortlut": 0,
+	"zlib_compression": "27",
+	"options": [ "led" ],
+	"contentids": [ 22, 23, 1, 2, 3, 4, 5, 7, 8, 9, 10, 11, 14, 16, 17, 18, 19, 20, 21, 27 ],
+	"usetemplate": 1
```

@nlimper @stevew817  Looking at the existing examples, I think a BWRY tag can simply refer the template for a BWR tag. If this is the case, then this template could be changed to:
```suggestion
	"usetemplate": 49
```
which links to the regular [M3 2.2 tag ](https://github.com/OpenEPaperLink/OpenEPaperLink/blob/bfff2ef0b9255f8a8df44e09133e227e68dc122e/resources/tagtypes/31.json)

---

Yeah, I didn’t realize the usetemplate field was used that way. I’m AFK until late Sunday, will fix the PR after I get back.